docs/packages/omc3/workflow.md

```diff
@@ -1,7 +1,220 @@
+
+For this `omc3` provides the `tbt_converter` entrypoint.
+A typical use consists in specifying the location of your turn-by-turn measurement files, the data type of said files and the location in which to write the converted `SDDS` data.
+The formats supported by the converter are:
+
+- Machine formats: `lhc`, `iota` and `esrf`, or any machine using one of these.
+- Simulation formats: the `trackone` and `ptc` formats from tracking data from `MAD-X` or `PTC`.
+
+??? info "The `trackone` and `ptc` formats"
+    What is referred to here as the `trackone` or `ptc` format is obtained by giving the `DUMP` and `ONETABLE` options to the `TRACK` or `PTC_TRACK` commands, respectively, in `MAD-X`.
+    This way, all tracking data is written to a single file on disk, which you can feed to the `tbt_converter`.
+    Remembering which tracking module was used is important as `PTC_TRACK` does not write in the same format as `MAD-X`'s `TRACK`.
+
+For our example, let us say that when setting up tracking we have asked `MAD-X` to output the data into a file called `trackone` (this is actually the default name).
+Using the converter to make a compatible `SDDS` file then goes as:
+```bash
+python -m omc3.tbt_converter \
+    --files trackone \
+    --tbt_datatype trackone \
+    --outputdir .
+```
+
+The converter will create new files with the `.sdds` suffix appended to the original filename.
+In our case, a `trackone.sdds` file will be created.
+
+!!! tip "Other Uses"
+    The `tbt_converter` provides additional functionality such as adding noise to the provided data or averaging across particles.
+    To use these options, refer to the [converter's API documentation][tbt_converter].
+
+## Creating a Model
+
+In order to perform the optics analysis, one needs a model of the given machine to compare to.
+For this, `omc3` provides the `model_creator` entrypoint, which allows you to run a model simulation of the desired machine and output the needed files.
+
+??? example "Supported Machines"
+    The out-of-the-box supported machines for model creation are `lhc`, `ps` and `psbooster`, machines we work on.
+    While the `skekb`, `JPARC`, `petra` and `iota` have accelerator classes, no model creator has been implemented for them yet.
+    It is possible to extend this list for your machine by defining an appropriate `Accelerator` class as well as a model creator.
+    See [this guide][new_machine_guide] for implementation steps.
+
+In our example, we would like to compare our data to the nominal model of the 2018 LHC. 
+Using the script to create a nominal model of the 2018 LHCB1, with the machine configuration defined in an `opticsfile`, goes as:
+```bash
+python -m omc3.model_creator \
+    --accel lhc \
+    --type nominal \
+    --year 2018 \
+    --beam 1 \
+    --energy 6.5 \
+    --nat_tunes 62.31 60.32 \
+    --modifiers path/to/opticsfile \
+    --outputdir lhc_model
```

Comment:
I think if we could have a complete example in the sense that it could be copy-pasted for anyone with access to AFS that would be great.

Reply:
I can point to an `opticsfile` on AFS yes, it wouldn't necessarily be the one a user wants but copy-pasting this would work by itself (with AFS access). Will do the change :+1: